packages/mongo/tests/hooks/direct.js

```
@@ -0,0 +1,81 @@
+import { Meteor } from 'meteor/meteor'
+import { Mongo } from 'meteor/mongo'
+import { Tinytest } from 'meteor/tinytest'
+
+function createTest (cname, conntype) {
+  Tinytest.addAsync(
+    `direct - update and remove should allow removing by _id string (${cname}, ${JSON.stringify(
+      conntype
+    )})`,
+    async function (test) {
+      // Use Mongo._collections instead of Mongo.Collection.get (no external package needed)
+      if (Mongo._collections.get(cname)) return
```

ESLint

[error] Insert `;`

(prettier/prettier)